index.html

```diff
@@ -15,23 +15,20 @@
     <div class="container-section">
         <div class="container-background">
             <div class="container-flex">
-                <a href="#" class="logo">My logo</a>
+              <div class="left-side">
+                <a href="#" class="logo link">My logo</a>
+              </div>  
+               <div class="right-side">
+                   <a href="#" class="top-link link">Portfolio</a>
+                   <a href="#" class="top-link link">About</a>
+                   <a href="#" class="top-link link">Services</a>
+               </div> 
                 <button type="button"><span class="material-icons-outlined">menu</span></button>
             </div>
             <div class="anim">
-                <div class="animation-div">
-                    <div></div>
-                    <div></div>
-                </div>
-                <div class="animation">
-                    <div></div>
-                </div>
-                <div class="animation-main">
-                    <div></div>
-                    <div></div>
-                </div>
+                
                 <div class="container">
-                    <p class="text-myName">I'm Arnaud<br>Glad to see you!</p>
+                    <h1 class="text-myName">I'm Arnaud<br>Glad to see you!</h1>
                     <p class="text-identification">I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
```

Comment:
- [ REQUIRED] Please fix the weird behavior that happens when you resize the website to the mobile version again